src/utils/web-client.js

```diff
@@ -6,9 +6,9 @@ const client = module.exports = Axios.create({
     headers: { common: {} }
 });
 
-client.updateConfiguration = ({ apiBaseUrl, ...rest }) => {
+client.updateAuth = ({ token }) => {
 
-    if (apiBaseUrl || apiBaseUrl === '') {
-        client.defaults.baseURL = apiBaseUrl;
-    }
+    const headers = client.defaults.headers.common;
+
+    return Object.assign(headers, { authorization: `Bearer ${token}` });
```

Comment:
I don't think this is being cleared on logout yet.

Reply:
Oh, okay. What needs to change for that to happen?

Reply:
This is something I recall doing together with that `try {} finally {}` block yesterday.

Reply:
Oh right! That can be found right here: https://github.com/BigRoomStudios/strangeluv/pull/241/files#diff-273dcc69b8aecc253010766ca5100d5dR44-R51